src/executionHandler.ts

```diff
@@ -2,16 +2,19 @@ import {
   IntegrationExecutionContext,
   IntegrationExecutionResult,
   IntegrationInvocationEvent
-} from '@jupiterone/jupiter-managed-integration-sdk';
+} from "@jupiterone/jupiter-managed-integration-sdk";
+
+import initializeContext from "./initializeContext";
 
 export default async function executionHandler(
   context: IntegrationExecutionContext<IntegrationInvocationEvent>
 ): Promise<IntegrationExecutionResult> {
+  const { graph, persister, provider } = await initializeContext(context);
+
+  const oldData = await graph.fetchEntities();
```

Comment:
I think this functions returns entities and relationships. Perhaps a better name would be:

```suggestion
  const oldData = await graph.fetchEntitiesAndRelationships();
```